crates/bevy_scene/src/dynamic_scene_builder.rs

```diff
@@ -63,6 +63,15 @@ impl<'w> DynamicSceneBuilder<'w> {
         self.extract_entities(std::iter::once(entity))
     }
 
+    pub fn remove_empty_entities(&mut self) -> &mut Self {
```

Comment:
```suggestion
    /// Despawns all enitities with no components.
    ///
    /// These were likely created because none of their components were present in the provided type registry upon extraction.
    pub fn remove_empty_entities(&mut self) -> &mut Self {
```